openshift/kustomize/overlays/openshift/olmv1-ns/patches/manager_deployment_node_selection.yaml

```
@@ -0,0 +1,25 @@
+- op: add
+  path: /spec/template/spec/nodeSelector
+  value: { "kubernetes.io/os": "linux", "node-role.kubernetes.io/master": "" }
+- op: add
+  path: /spec/template/spec/tolerations
```

Will this override tolerations if they are already present from upstream? Should we instead use the append syntax?
```suggestion
  path: /spec/template/spec/tolerations/-
```

---

Append syntax failed because it didn't already exist. Might override tolerations already present from upstream but that should be fine for now as we are configuring them for what we think we need for openshift